js/noms/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@attic/noms",
   "license": "Apache-2.0",
-  "version": "67.1.0",
+  "version": "68.0.0",
```

Comment:
Note I bumped the major version since the semantics changed on ValueBase.walkValues